src_c/include/_pygame.h

```diff
@@ -64,7 +64,7 @@
     (PG_VERSIONNUM(PG_MAJOR_VERSION, PG_MINOR_VERSION, PG_PATCH_VERSION) >= \
      PG_VERSIONNUM(MAJOR, MINOR, PATCH))
 
-#include "pgcompat.h"
+#include "../pgcompat.h"
```

Comment:
This scares me to my core
/s
But only kinda

Reply:
i couldnt find another way to include `src_c/pgcompat.h` 
if you have any suggestions about how i should do that please tell me